### PR TITLE
Corrected menu items and section headers

### PR DESCRIPTION
**Pull request by Ruslan-Aleev**
### What does it do?
Corrected menu items and section headers, buttons.
Corrected some inaccuracies in the menu: deleted points, somewhere there were points in the description, but somewhere not.
Corrected some inaccuracies in the import section: the buttons used the lexicon from the menu topic, and not the lexicon from the import topic, it was wrong.

### Related issue(s)/PR(s)
https://github.com/modxcms/revolution/issues/14457

### CODE REVIEW

**Comment by Ruslan-Aleev** [comment]
It seems to me that the item "Extras" is more correctly to call "Packages", because In this section, the packages will be installed.
But at the same time, in the description of the section "Namespaces", the "Add-ons" should be called as "Components", because in this section, the components are located (installed packages).

File: core/lexicon/en/topmenu.inc.php [hunk 1]
```diff
@@ -17,7 +17,7 @@
 $_lang['api_docs_desc'] = 'Complete API documentation for MODX Revolution.';
 $_lang['bespoke_manager'] = 'Manager Customization';
 $_lang['bespoke_manager_desc'] = 'Manage bespoke MODX Managers';
-$_lang['components'] = 'Extras';
+$_lang['components'] = 'Packages';
```
Comment (opengeek) [inline core/lexicon/en/topmenu.inc.php:20]: No. Extras are what could be installed by a package, but they are not the same thing.

**Review by opengeek — CHANGES_REQUESTED** [review]
These commits include a lot of miscellaneous changes that should not be combined in my opinion.

File: core/lexicon/en/topmenu.inc.php (hunk 1 above)
Comment (Ruslan-Aleev) [reply core/lexicon/en/topmenu.inc.php:20]: Why so complicate? We can leave only "Packages" and "Components" - installed packages.
Why need for the entities "Extras" and "Add-ons"?

Comment (opengeek) [reply core/lexicon/en/topmenu.inc.php:20]: Add-Ons/Extras/Components <-- these are interchangeable and different than packages. I'm fine with picking one of those, but replacing any of those with packages is simply wrong. It's not complicated at all.

Comment (opengeek) [reply core/lexicon/en/topmenu.inc.php:20]: The more important issue is these things need to be discussed independently from all the miscellaneous changes included in this single PR.

Comment (Ruslan-Aleev) [reply core/lexicon/en/topmenu.inc.php:20]: Issue hang in 2017, for example, https://github.com/modxcms/revolution/issues/13554 but there is no progress in the discussion.

**Comment by JoshuaLuckers** [comment]
Changes are more likely to be accepted if you make a new PR for each "group" of changes. Especially when there are changes that need discussion. I believe it's also more clear what the definition of a "Extra" is.

Thanks again for helping make MODX better @Ruslan-Aleev 😎 !

**Comment by Ruslan-Aleev** [comment]
@JoshuaLuckers Yes, about several groups that I have in vain combined - no questions, it is worth splitting into separate PRs.
But there is a question to the wording of @opengeek _"these things need to be discussed"_, there is a discussion, but Code Owners do not participate in the discussion, and even if they participate, the result of the discussion is not clear.
For example, here https://github.com/modxcms/revolution/issues/14291 there is a structure, there is a discussion, but there is no result of the discussion. We are waiting for more discussion or what? If there was a specific comment from Owners, we do this and that, or we leave it this way and that - this is the result, but instead we get a discussion for discussion. What is not productive, in my opinion.
Owners probably have a lot of work and do not have time to clearly state their position in the comments, but there are issue that have been hanging since 2009 and this is depressing.

**Pull request closed without merging**